networkx/algorithms/core.py

```diff
@@ -69,6 +69,13 @@ def core_number(G):
         The k-core is not implemented for graphs with self loops
         or parallel edges.
 
+    Examples
+    --------
+    ... e = [(0, 1), (0, 2), (0, 3), (1, 2), (2, 3), (3, 4), (4, 5), (4, 6)]
+    ... G = nx.Graph(e)
+    ... nx.core_number(G)
+    {0: 2, 1: 2, 2: 2, 3: 2, 4: 1, 5: 1, 6: 1}
```

Comment:
Thanks for the PR @PurviChaurasia!

This is not the right way to write python codeblocks in the docs, could you look at other examples and use the correct syntax (>>>).
You can look at how this looks for our generated documentation at https://output.circle-artifacts.com/output/job/9821ef59-e80b-4210-b47b-d16f63efb9ad/artifacts/0/doc/build/html/reference/algorithms/generated/networkx.algorithms.core.core_number.html#networkx.algorithms.core.core_number

You can find these generated docs at the end of the PR, click on "show all checks" and then click on details next to ci/circleci: documentation artifact — Link to 0/doc/build/html/index.html

Reply:
Sure thanks! Will change.

Reply:
@PurviChaurasia Hi! How did you make sure that the output was coming right under >>> code block and passing the tests? It is not passing tests if I pass in a >>> code block since it is returning output where it shouldn't.

Reply:
Hi @ManasviEmmadi the code is supposed to be under >>> block. The output however comes under it and is read as such. I think I'm not understanding your question properly, perhaps you can add a few screenshots?

Reply:
![image](https://user-images.githubusercontent.com/93079732/226155306-1ba1f8d2-b7b6-4a82-b3e8-6c347519b9c3.png)
So I have written it like this and when I run it, this is my output:
![image](https://user-images.githubusercontent.com/93079732/226155375-158844ba-9b5a-442b-b28d-49d756ac40b7.png)
I do not seem to get the output under the code block like that.

Reply:
The output is also supposed to be added by you yourself below the code block. Hence whatever output you are getting paste that below.

Reply:
Thanks!